src/persist/src/nemesis/direct.rs

```diff
@@ -60,6 +80,11 @@ impl Runtime for Direct {
                 Res::StorageAvailable
             }
         };
+
+        // Poke the dataflows a bit. We really only need the one in step (and
```

Comment:
I'm not sure what "step" means where it says:

"We really only need the one in step (and stop)"

is that meant to be "seal" or something else?